### PR TITLE
Initial frontend for ChatGPT integration

### PR DESCRIPTION
**Pull request by ryanclark**
WIP

### CODE REVIEW

File: web/packages/teleport/src/Assist/Chat/Chat.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  }, [ref.current]);
+
+  useEffect(() => {
+    scrollTextarea();
+  }, [messages, scrollTextarea]);
+
+  const handleSubmit = useCallback(
+    (message: string) => {
+      send({
+        author: Author.User,
```
Comment (xacrimon) [inline web/packages/teleport/src/Assist/Chat/Chat.tsx:84]: I don't think we need to specify this here. We'll handle it in the backend so you can't go bananas with it. And you shouldn't be able to send as anything other than yourself either.

Comment (ryanclark) [reply web/packages/teleport/src/Assist/Chat/Chat.tsx:84]: Yeah, this is just left over from my backend. It needs to change once the spec for what needs to go to the backend is decided on.

Comment (ryanclark) [reply web/packages/teleport/src/Assist/Chat/Chat.tsx]: I've changed this to just publish the string of the message to the websocket

**Review by jakule — APPROVED** [review]
LGTM. Great job 🎉 :shipit:

**Pull request merged**